docs/maintain/maintain-sync.md

```diff
@@ -196,6 +197,87 @@ example:
 
 - Find your node on [Telemetry](https://telemetry.polkadot.io/#list/Polkadot)
 
+</TabItem>
+<TabItem value="linux-package">
+
+You can also install Polkadot from one of our package repositories.
+
+Installation from the Debian or rpm repositories will create a `systemd`
+service that can be used to run a Polkadot node. The service is disabled by default,
+and can be started by running `systemctl start polkadot` on demand (use
+`systemctl enable polkadot` to make it auto-start after reboot). By default, it
+will run as the `polkadot` user.  Command-line flags passed to the binary can
+be customized by editing `/etc/default/polkadot`. This file will not be
+overwritten on updating polkadot.
+  
+### Debian-based (Debian, Ubuntu)
+
+Currently supports Debian 10 (Buster) and Ubuntu 20.04 (Focal), and
+derivatives. Run the following commands as the `root` user.
+
+```bash
+# Import the security@parity.io GPG key
+gpg --recv-keys --keyserver hkps://keys.mailvelope.com 9D4B2B6EB8F97156D19669A9FF0812D491B96798
```

Comment:
Hey @DrW3RK, this PR came in last week and I wanted to ask you about some of the technical details.  I am not very familiar with this setup process.  Is there someone else you would recommend for additional review?